Added getPlayerInfo function

Neztore opened the pull request:
Added the getPlayerinfo function

Required arguments: playerID

**Returns:**
        username: Username of target
        status: Status of target
        blurb: blurb of target
        joinDate: Join date of target (Date object)
        age: Days since the target user joined ROBLOX.

This is for #56

Commit:
Added getPlayerInfo function

bbsquade101 commented:
Thank's Nez it helped a lot!

suufi commented:
Age is redundant
joinDate should return Date object

sbhadr commented:
@Neztore 
Age and JoinData are virtually the same.
Age calculates how long it's been since said player has joined and JoinDate is the date of which said player joined.
I'd leave it up to the person using the lib to calculate their "age" rather than providing it.

Commit:
added date parsing

sentanos commented on lib/user/getPlayerInfo.js:
```diff
@@ -0,0 +1,42 @@
+      let username = body('.header-title h2').text();
+      let joinDate = body('.profile-stats-container .text-lead').text().slice(0,9);
+
+      joinDate = Date.parse(joinDate);
```
For consistency throughout the project please use [rbx.getDate](https://github.com/sentanos/roblox-js/blob/master/lib/util/getDate.js), which accepts a time zone (and handles daylight savings), and use ROBLOX's main time zone (which is central time). Although this isn't an exact timestamp and it's hard to check which time zone it actually is (ROBLOX can be inconsistent too), the best bet is to just use the same thing as everything else.

Commit:
Made joinDate() use util.getDate()

Neztore commented:
@sentanos

clockworksquirrel committed:
Added Avatar, Chat, and Party features

Commit:
Fixed the joinDate so that it is always correct

Neztore commented:
@Sloss2003 If you need it for a project, see [this](https://github.com/Neztore/roblox-js) fork

mwalden2004 commented:
@Neztore I made it myself before i saw it haha

Commit:
Delete forumPost.js

Commit:
Deleted all forum related functions

ROBLOX removed the forum. These are no longer relevant.

Commit:
Merge pull request #1 from ClockworkSquirrel/master

Added Avatar, Chat, and Party features

clockworksquirrel committed:
Developer Products and AssetVersionIds

Adds features relating to Developer Products and Asset Versions.

Developer Products can now be created and updated via addDeveloperProduct() and updateDeveloperProduct().

The current VersionId of assets can now also be retrieved via getAssetVersionId().

Commit:
Merge pull request #3 from ClockworkSquirrel/master

Developer Products and AssetVersionIds

Commit:
First commit to match noblox
